pullbug/github_bug.py

```diff
@@ -122,7 +123,9 @@ def prepare_message(cls, pull_request):
         # Truncate description after 120 characters
         description = (pull_request['body'][:120] + '...') if len(pull_request
                                                                   ['body']) > 120 else pull_request['body']
-        message = f"\n:arrow_heading_up: *Pull Request:* <{pull_request['html_url']}|" + \
-            f"{pull_request['title']}>\n*Description:* {description}\n*Waiting on:* {users}\n"
+        repository_name = (pull_request['repo'][0]['name'])
+
+        message = f" *Repository:* {repository_name}\n:arrow_heading_up: *Pull Request:* <{pull_request['html_url']}|" + \
```

Comment:
Looks pretty good, you'll want to keep the structure the same though (array up emoji should be first).